README.md

```diff
@@ -0,0 +1,58 @@
+# Light Wallet Protocol Canonical files
+
+This repository contains the canonical Protobuf definitionss of the Light Wallet protocol.
+
+## Rationale
+Before the creation of this repository, copies of wallet RPCs `.proto` files would exist in several
+repositories across the Zcash ecosystem. This posed a challenge for protocol developers,
+wallet maintainers and infrastructure providers who would have to look up for the _Ad Hoc_
+canonical files for their own use and changes would be hard to propagate and communicate.
+
+This repository IS NOT a specification of the Zcash Light Client protocol. The Zcash Light Client
+Protocol is described in [ZIP-307](https://zips.z.cash/zip-0307).
+
+These files are the Protocol Buffers used in that ZIP using [proto 3](https://protobuf.dev/programming-guides/proto3/).
+
+## How to use these files
+
+We recommend using `git subtree` of the `main` branch  or tagged versions of this repository.
+
+```git subtree -p lightwallet-protocol/ pull git@github.com:zcash/lightwallet-protocol.git main --squash```
+
+
+### Example: YourProject
+We assume YourProject is a git repository
+
+````
+$ cd YourProject
+
+$ git subtree -p lightwallet-protocol/ pull git@github.com:zcash/lightwallet-protocol.git main --squash
+
```

Comment:
Personally I think these empty lines don't add anything, should be removed. Just a suggestion.